lib/observer/src/etop_defs.hrl

```diff
@@ -19,14 +19,14 @@
 %% 
 %% %CopyrightEnd%
 %%
--define(SYSFORM,
-	" ~-72w~10s~n"
-	" Load:  cpu  ~8w               Memory:  total    ~8w    binary   ~8w~n"
-	"        procs~8w                        processes~8w    code     ~8w~n"
-	"        runq ~8w                        atom     ~8w    ets      ~8w~n").
+-define(SYSFORM(IsTerminal, ShellMode),
+	" ~-72w~10s" ++ newline(IsTerminal, ShellMode) ++
+	" Load:  cpu  ~8w               Memory:  total    ~8w    binary   ~8w" ++ newline(IsTerminal, ShellMode) ++
+	"        procs~8w                        processes~8w    code     ~8w" ++ newline(IsTerminal, ShellMode) ++
+	"        runq ~8w                        atom     ~8w    ets      ~8w" ++ newline(IsTerminal, ShellMode)).
```

Comment:
Move this to etop_txt.erl ?

I don't like function calls in macros, if they can't be reused in other files it should not be there..

Reply:
How about changing this newline function to macro then?

Reply:
Moved macro to etop_txt.erl